psutil/_psosx.py

```diff
@@ -212,6 +214,31 @@ def disk_partitions(all=False):
 # =====================================================================
 
 
+def sensors_temperatures():
+    """Return hardware (CPU and others) temperatures as a dict
```

Comment:
doesn't this return temperatures about (1) CPU only?

Reply:
You are right, it could be possible to add additional sensors (via SMC reads) in the future